releases/latest_version.go

```diff
@@ -27,6 +27,7 @@ type LatestVersion struct {
 	InstallDir         string
 	Timeout            time.Duration
 	IncludePrereleases bool
+	Enterprise         EnterpriseOptions
```

Comment:
As mentioned we could turn this into a pointer, which could be equivalent to `Enterprise == true` but slightly simpler.